validator/bundler.py

```diff
@@ -99,16 +99,18 @@ def fix_dir(directory):
 @click.argument('data-dir', type=click.Path(exists=True))
 @click.argument('resource-dir', type=click.Path(exists=True))
 @click.argument('git-commit')
+@click.argument('git-commit-timestamp')
 def main(resolve, thread_pool_size,
          schema_dir, graphql_schema_file, data_dir, resource_dir,
-         git_commit):
+         git_commit, git_commit_timestamp):
     schema_dir = fix_dir(schema_dir)
     data_dir = fix_dir(data_dir)
     resource_dir = fix_dir(resource_dir)
 
     bundle = {}
 
     bundle['git_commit'] = git_commit
+    bundle['git_commit_timestamp'] = git_commit_timestamp
```

Comment:
shouldn't we check the timestamp format somewhere?

Reply:
Maybe using a UNIX timestamp would help to avoid the format hell https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/12222